invenio_files_rest/models.py

```diff
@@ -1111,6 +1121,112 @@ def relink_all(cls, old_file, new_file):
             ObjectVersion.query.filter_by(file_id=str(old_file.id)).update({
                 ObjectVersion.file_id: str(new_file.id)})
 
+    def get_tags(self):
```

Comment:
Because I copy/paste the behavior from [BucketTags](https://github.com/inveniosoftware/invenio-files-rest/blob/master/invenio_files_rest/models.py#L399) 😉